### DISCUSSION TITLE
人工智慧結合金融科技 中小微企信保融資服務平台首發

### DISCUSSION THREAD

**Discussion by kaoyihua27** [1101-GS3073-C/Python-Community-GS3073-C#152] [General]
https://www.google.com/amp/s/www.chinatimes.com/amp/realtimenews/20211223005778-260410

這個挺適合的欸~

**Comment by bonny123865** [comment]
這種合作開發出來的平台不只可以讓企業跟銀行快速的處理資金問題，而且零接觸也是防疫期間很棒的一個特性！

**Comment by briankuo123** [comment]
這對規模比較小的企業真的是一大福音。許多企業可能在剛起步的時候或要擴大時就被高到嚇死人的成本逼退了。

**Comment by minniewei** [comment]
但若處理不好，可能就會變成另類的地下錢莊了!!!

**Comment by wh20502** [comment]
這個技術的主要問題我覺得是法律責任，感覺還沒有相關規定，可能要等之後有事情才會更新。此外我覺得以後可能會發展個人信用點數，之類的機制幫忙AI審查。

**Comment by ghost** [comment]
跟錢有關感覺就會牽扯好多東西
